.github/workflows/release-publish.yml

```diff
@@ -84,7 +84,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        base-image: [ alpine, centos ]
+        include:
+          - base-image: alpine
+            net-version: net8.0
+          - base-image: centos
+            net-version: net7.0
```

Comment:
Can you please describe why?

Reply:
.NET8 is not supported on Centos.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/actions/runs/6953620569/job/18919049333

I see here: https://learn.microsoft.com/en-us/dotnet/core/install/linux-centos#supported-distributions 👍